guacamole-ext/src/main/java/org/apache/guacamole/token/StandardTokens.java

```diff
@@ -42,6 +43,16 @@
     public static final String PASSWORD_TOKEN = "GUAC_PASSWORD";
 
     /**
+     * The name of the client token added via addStandardTokens().
+     */
+    public static final String REMHOST_TOKEN = "GUAC_REMHOST";
```

Comment:
I'm fine with that...I think the reason I settled on the shorter ones is I actually ran into an issue with a limit on the length of the tokens??  Can't remember exactly off the top of my head, but I think even GUAC_HOSTNAME and GUAC_ADDRESS were too long, but never fully tracked it down.  I'll try, again.